src/components/FormInput.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
- Optional
Consider renaming this file to something like `CreateNewBook` . `FormInput` is too generic a name. Once the application gets large and you have multiple forms for different sections of the application, it would be difficult to manage the application.

Reply:
Thank you, sir, for raising my attention to this point.